zossrc/chdr/wtoexec.h

```diff
@@ -0,0 +1,31 @@
+#ifndef WTOEXEC_H
+#define WTOEXEC_H
+
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+
+/**                                                                                         
+ * C/C++ interlanguage - "OS" meaning to call non-Language Environment conforming assembler.
+ */
+#if defined(__cplusplus) && (defined(__IBMCPP__) || defined(__IBMC__))
```

Comment:
It's fine to keep this stuff in here, but some of this preprocessor if logic was to allow compilation off z/OS. I'm just not sure if it confuses the example?

Reply:
I agree that it confuses the example.  It seems that you need some level of #ifdef for the metal C compilation of the header and for the C++ header.  If I'm leaving some, perhaps it's best to leave this as-is to get people setup for on and off platform compilation.  What do you think?

Reply:
Sounds good to me.